### ISSUE TITLE
Consider removing Default impl and replacing it with a list of preset configs

### ISSUE THREAD

**Issue by clarcharr** [brayniac/ratelimit#21]
The `Default` impl doesn't make too much sense because most use cases will need to specify what default makes sense. While one API might want to limit every second, another may want to limit every millisecond, etc.

**Comment by brayniac** [comment]
I agree the Default impl doesn't make much sense. What would a reasonable list of configs look like? Should we address this in the code? Or just in docs?
